server/services/userService.ts

```diff
@@ -1,38 +1,13 @@
-import { jwtDecode } from 'jwt-decode'
-import { convertToTitleCase } from '../utils/utils'
-import type { User } from '../data/manageUsersApiClient'
-import ManageUsersApiClient from '../data/manageUsersApiClient'
-import type { HmppsAuthClient, NomisUserRolesApiClient, PrisonApiClient, RestClientBuilder } from '../data'
+import type { HmppsAuthClient, PrisonApiClient, RestClientBuilder } from '../data'
 import logger from '../../logger'
 
-export interface UserDetails extends User {
-  displayName: string
-  roles: string[]
-}
-
+// TODO review and probably remove class
```

Comment:
Once 'Change establishment' page goes, this class and the support for the APIs it calls can go - as all this functionality will be done via DPS header.